oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst

```diff
@@ -122,7 +123,9 @@ References
     spec:
 
       # (Required) This can be a group(`group:<group_name>` or a user(`user:<github_username>`)
-      owner: group:docs.edx.org-maintainers
+      # Don't forget the "user:" or "group:" prefix, and group names need their
+      # organization included: "group:openedx/devs", not "group:devs".
+      owner: group:openedx/docs.edx.org-maintainers
```

Comment:
@feanil do we agree that the organization is needed? You said the group is actually a name in Backstage, not a GitHub team name.

Reply:
The group name (and user name come to think of it) have to correlate with something. I can't write `group:nedbat-fictional-team-xyzzy` in this field.  What do the names have to correspond with?

Reply:
It's just github team names or user names prefixed with either `user` or `group`.

Reply:
Must it be a team name in the `openedx` org?

Reply:
Yes, it must be a team name in the `openedx` org for it to be properly linked.